src/main/resources/config/config.yaml

```diff
@@ -17,6 +17,7 @@ logging:
 
 baseUrl: ${PUBLICAPI_BASE}
 connectorUrl: ${CONNECTOR_URL}
```

Comment:
Maybe we should rename:
- `connectorUrl -> connectorCCUrl`
- `CONNECTOR_URL -> CONNECTOR_CC_URL`
as part of clean up or refactoring in the future